### PR TITLE
Support for k8s version 1.16

### PR DESCRIPTION
**Pull request by radut**
Deprecation apis : https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Signed-off-by: Radu Toader <radu.m.toader@gmail.com>

### CODE REVIEW

**Comment by radut** [comment]
https://github.com/cloudworkz/kube-eagle-helm-chart/issues/21

File: kube-eagle/templates/ingress.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "kube-eagle.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```
Comment (weeco) [inline kube-eagle/templates/ingress.yaml]: I think they have migrated from extensions/v1beta1 to networking.k8s.io/v1beta1 now (with v1.14 only though), and therefore I'd like to keep it for some more time like it is.

Comment (radut) [reply kube-eagle/templates/ingress.yaml]: I don't really understand how you want it to be. This PR is for supporting only 1.16 onwards, I will change there too the version to : networking.k8s.io/v1beta1

Comment (weeco) [reply kube-eagle/templates/ingress.yaml]: To my knowledge there's no `extensions/v1` api. They are migrating to `networking.k8s.io/v1beta1` in Kubernetes 1.16 (which has been introduced in Kubernetes v1.14). `extensions/v1beta1` will be deprecated with 1.20+, that's why I suggested to leave the apiVersion like it was until 1.20 is actually released.

See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#ingress-v1beta1-networking-k8s-io and https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#ingress-v1beta1-extensions

**Comment by radut** [comment]
Ok. Ill compare with master branch and pick that value

**Comment by weeco** [comment]
Perfect thanks for the PR

**Pull request merged**